CHANGELOG/CHANGELOG-0.10.md

```diff
@@ -1,3 +1,63 @@
+## v0.10.6
```

Comment:
I see there's a merge commit here, usually I favor rebases over merges, but if the branch is too wonky to do that then this is fine

Reply:
Let me know if you figure it out. All the go mod stuff we did for openshift makes it quite a painful process.

Reply:
yeah The majority of the patches applied cleanly but the first didn't and it's not clear to me if it would. I think this works